### ISSUE TITLE
Check failed: cusolverDnCreate(&cusolver_dn_handle) == CUSOLVER_STATUS_SUCCESS Failed to create cuSolverDN instance.

### ISSUE THREAD

**Issue by Lee-1031** [rpautrat/SuperPoint#335]
I rented a GPU from AutoDL and used the TensorFlow framework to train a deep learning model. I carried out repeated experiments using a 'for' loop. Within each layer of the loop, I trained a FNN and estimated the parameter beta. Finally, I concatenated the obtained betas. However, I encountered the following problems: 

2025-02-20 00:02:34.409341: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F AVX512_VNNI FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2025-02-20 00:02:34.556319: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1532] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 22182 MB memory:  -> device: 0, name: NVIDIA GeForce RTX 4090, pci bus id: 0000:3d:00.0, compute capability: 8.9
2025-02-20 00:02:38.344381: I tensorflow/stream_executor/cuda/cuda_blas.cc:1786] TensorFloat-32 will be used for the matrix multiplication. This will only be logged once.
Epoch 100, Loss: -48.925506591796875
Epoch 200, Loss: -104.349609375
4/4 [==============================] - 0s 1ms/step
2025-02-20 00:10:45.682877: I tensorflow/core/util/cuda_solvers.cc:179] Creating GpuSolver handles for stream 0x55ba1062a560
2025-02-20 00:10:46.251520: F tensorflow/core/util/cuda_solvers.cc:114] Check failed: cusolverDnCreate(&cusolver_dn_handle) == CUSOLVER_STATUS_SUCCESS Failed to create cuSolverDN instance.

Process finished with exit code 134

I've tried this solution but there is no help :(
physical_devices = tf.config.list_physical_devices('GPU')
try:
    tf.config.experimental.set_memory_growth(physical_devices[0], True)
except:
    pass

How to solve the problem? Much thanks!

**Comment by Lee-1031** [comment]
This issue has been resolved. I rented the RTX 4090 and chose the latest version of TensorFlow, but the GPU and TensorFlow version were too "advanced." So, I opted for an older but more stable GPU and TensorFlow, and the problem was solved just like that.
